src/sap/fhir/model/r4/FHIRModel.js

```diff
@@ -1531,5 +1531,25 @@ sap.ui.define([
 		throw new Error("Unsupported operation: sap.fhir.model.r4.FHIRModel#getContext");
 	};
 
+	/**
+	 * Determines the Structure Definition URL for a given binding info (meta.profile)
+	 * Default URL would be base profile URL + resource type
+	 *
+	 * @param {sap.fhir.model.r4.lib.BindingInfo} oBindingInfo The binding info object
+	 * @returns {string} The structure definition for the given binding info
+	 * @public
```

Comment:
change visibility to protected. Currently public availability isn't needed

Reply:
done